server/src/initialize.rs

```diff
@@ -144,7 +144,7 @@ pub fn init_wizard(conf: &ServerConfig, opts: InitializeOpts) -> Result<(), Erro
         Input::with_theme(&theme)
             .with_prompt("Network CIDR")
             .with_initial_text("10.42.0.0/16")
-            .interact()?
+            .interact_text()?
```

Comment:
Optionally we could add a TODO here and below mentioning we work-around the issue (and link it), mention it limits to ASCII.